rules/opentitan/defs.bzl

```diff
@@ -231,6 +231,7 @@ def opentitan_test(
             rsa_key = rsa_key,
             spx_key = spx_key,
             manifest = manifest,
+            need_jtag = getattr(tparam, "need_jtag", False),
```

Comment:
I think this is the part that I am missing?

Reply:
Yes, but I think you were also missing the first change in `exec_env.bzl` (adding fields to `_FIELDS` so that they get copied to the exec env provider)